tools/Utilities/EnvVariablesUtility/TelemetryEvents/EnvironmentVariablesProfileEnabledEvent.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using DevHome.Telemetry;
+using Microsoft.Diagnostics.Telemetry;
+using Microsoft.Diagnostics.Telemetry.Internal;
+
+namespace DevHome.EnvironmentVariables.TelemetryEvents;
+
+[EventData]
+public class EnvironmentVariablesProfileEnabledEvent : EventBase
+{
+    public override PartA_PrivTags PartA_PrivTags => PrivTags.ProductAndServiceUsage;

```

Comment:
If this event is **not** from a user action, please change the PrivTags to `ProductAndUsagePerformance`

Reply:
Same for all other Telemetry classes.

Reply:
This is result of user updating env variable. So should be PSU right?

Reply:
@dkbennett

Reply:
This isn't the recording the actual act of the user pushing the button, so it's Performance.

Reply:
Fixed